src/pages/Lend/LendByCategoryPage.vue

```diff
@@ -137,7 +150,7 @@ export default {
 			categorySetting: [],
 			categorySetId: '',
 			itemsInBasket: [],
-			imageEnhancementExperimentVersion: '',
+			imageEnhancementExperimentVersion: null,
```

Comment:
I'll finish cleaning this up (including commented code below) later in the sprint.